Gemfile

```
@@ -3,3 +3,4 @@ gem 'jekyll', '4.2.0'
 gem 'kramdown-parser-gfm'
 gem 'jekyll-redirect-from'
 gem 'rouge', '3.26.0'
+gem 'liquid-c'
```

```suggestion
gem 'liquid-c'

```
Adding a newline to appease the linter.